### ISSUE TITLE
How does SoftU2FTool deal with multiple devices

### ISSUE THREAD

**Issue by anglinb** [github/SoftU2F#2]
**tl;dr** Haven't registered my SoftU2FTool with GitHub and I tried to use my Yubikey but it doesn't work because (I believe) SoftU2FTool is intercepting the requests or I'm messing up the flow somehow.

---
 
I need to investigate further to understand what is actually happening but what I experienced:

1. Installed SoftU2FTool
2. Needed to enable _sudo mode_ on GitHub by entering my password or using my Yubikey. I got the "Register with site"
    <img width="375" alt="banners_and_alerts" src="https://user-images.githubusercontent.com/2637602/27756243-cf523e90-5daa-11e7-9981-ef0ff26e2c89.png">
3. When I tried to use my Yubikey it entered in the password field, as if I had just pressed the key in a TextEdit window or something.

**Thoughts on how to remedy this behavior:**

1. We could add a way to enable or disable the SoftU2FTool? 
2. We could add some sort of fallback logic if SoftU2FTool doesn't find a record for the site that is sending the request.
3. I could just move over everything to SoftU2FTool 😎

**Comment by btoews** [comment]
Weird. I can't reproduce this at all. You don't see the LED on your Yubikey flashing? FWIW, a good site for testing U2F stuff is https://demo.yubico.com/u2f.

**Comment by btoews** [comment]
I'm closing this for now because I can't reproduce the issue. Feel free to reopen the issue if you're still having trouble.
